WyattHWilliams/react-challenge-project-jan-2022
# fix(order-form): change set state from event to event.target

## Changes
1. change setting menu item and quantity from event.value to event.target.value

## Purpose
address [issue 1](https://github.com/Shift3/react-challenge-project-jan-2022/issues/1)

## Approach
sets the state to look at the right object param

## Testing Steps
1. select a menu item and quantity, submit form
2. check view orders page for new item

Closes #001
